pyccel/stdlib/math/pyc_math_f90.f90

```diff
@@ -31,6 +31,11 @@ module pyc_math_f90
     module procedure amax_01
 end interface
 
+interface amin
+    module procedure amin_00
+    module procedure amin_01
+end interface
+
```

Comment:
In order to be consistent with the naming convention in this file, we should use `_4` instead of `_00` and `_8` instead of `_01`. The number `4` or `8` identifies the datatype `precision` in Pyccel. This applies to both `amin` and `amax`.